openshift/e2e-tests-openshift.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/sh
+
+source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
+
+set -x
+
+readonly API_SERVER=$(oc config view --minify | grep server | awk -F'//' '{print $2}' | awk -F':' '{print $1}')
```

Comment:
`oc config view --minify | awk -F: '/server:/ { gsub(" http.://", ""); print $2}'` may be simpler